package.json

```diff
@@ -56,6 +57,7 @@
     "@babel/preset-flow": "^7.0.0",
     "@babel/register": "^7.0.0",
     "autoprefixer": "^7.1.1",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
what is this babel core for?

Reply:
Oh, there is another babel-core there. I thought I removed it in my last commit. I've added that version because it's advised on the jest documentation. It doesn't seem to work with the latest version.
https://jestjs.io/docs/en/getting-started#using-babel